### PR TITLE
add gcp support for arm64 upgrade jobs

### PR DESCRIPTION
**Pull request by lwan-wanglin**
add gcp support for arm64 upgrade jobs

### CODE REVIEW

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-arm64-nightly-4.16-upgrade-from-stable-4.15-gcp-ipi-disc-priv-oidc-f14

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-arm64-stable-4.15-upgrade-from-stable-4.14-gcp-ipi-proxy-etcd-encryption-f28

**Comment by lwan-wanglin** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-arm64-nightly-4.15-upgrade-from-stable-4.14-gcp-ipi-disc-priv-oidc-f14

**Comment by lwan-wanglin** [comment]
I see there's no gcp platform in arm single arch upgrade jobs, so adding it. 
@JianLi-RH @shellyyang1989 @jianlinliu could you help review the pr? thanks!

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-stable-4.15-upgrade-from-stable-4.14.yaml [hunk 1]
```diff
@@ -139,6 +139,20 @@ tests:
...
+- as: gcp-ipi-proxy-etcd-encryption-f28
+  cron: 11 10 28 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:arm64-target
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      ENABLE_OTA_TEST: OCP-69948
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-stable-4.15-upgrade-from-stable-4.14.yaml:151]: I guess we do not need this line.

Comment (lwan-wanglin) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-stable-4.15-upgrade-from-stable-4.14.yaml:151]: what's the case for? I see the test case are enabled in amd64 files. https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-4.16-upgrade-from-stable-4.16.yaml#L91

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-stable-4.15-upgrade-from-stable-4.14.yaml:151]: This is for specific OTA test case coverage, for the newly added jobs, does not need it. If needed, OTA team should be responsible for updating the job config.

Comment (lwan-wanglin) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-stable-4.15-upgrade-from-stable-4.14.yaml]: Got it, thanks.

**Comment by jianlinliu** [comment]
@liangxia to review.

**Pull request merged**